### DISCUSSION TITLE
Text Surface class - what is it and why would we use it?

### DISCUSSION THREAD

**Discussion by MacelinCane** [mdbootstrap/TW-Elements#2348] [Support from community]
Hi All,
I have a project using Tailwinds Elements.
 
Quite often, in examples, there are classes: _text-surface_ and _bg-surface_.
We've never heard of 'surface' and it appears in ZERO documentation from either library provider (TWE or Tailwindcss), but searching google yields little help either.

Super curious about this. Any help would be greatly appreciated!

**Comment by iprzybysz** [comment]
Hi @MacelinCane , `surface` is a custom style which was added by us as part of our theme and is declared in the [plugin.cjs](https://github.com/mdbootstrap/TW-Elements/blob/6616b67d7e1cdc9d67999f55585c51a57e5dd90e/plugin.cjs#L762) file. For more information on adding custom styles in Tailwind CSS, please refer to the [Tailwind CSS Adding custom styles documentation](https://tailwindcss.com/docs/adding-custom-styles).

**Comment by MacelinCane** [comment]
Hi,
Thanks!  Can you add where I can find documentation on it? I'm not talking about in the code where I need to decipher what colors or whatever it is, but something that actually shows us something we can reference in order to keep using it? I've so far had to manually purge it from my project elements because I can find no reference as to which purpose is serves. Something like this perhaps.... https://tailwindcss.com/docs/customizing-colors
Thanks!

**Comment by iprzybysz** [comment]
Unfortunately, we don't have such documentation. Still, I recommend you use the [Tailwind CSS IntelliSense extension](https://tailwindcss.com/docs/editor-setup#intelli-sense-for-vs-code) in your code editor which will show you all available classes in the project while writing the code.
